packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -30,19 +30,6 @@ const propTypes = {
    * The container to wrap the disclosed content. This should be provided from the application level.
    */
   withDisclosureContainer: PropTypes.func,
-  /**
-   * If set to true, then the focus lock will get enabled.
-   */
-  shouldTrapFocus: PropTypes.bool,
-  /**
-   * If set to true, then the outside click will get enabled.
-   */
-  closeOnOutsideClick: PropTypes.bool,
```

Comment:
This change will break MPages

Reply:
Communicated: These changes are requirements for v4. Meanwhile this is being done so that v3 consumers do not break with focus trapping.